pkgs/os-specific/linux/nftables/default.nix

```diff
@@ -26,7 +26,7 @@ stdenv.mkDerivation rec {
   buildInputs = [
     libmnl libnftnl libpcap
     gmp jansson readline
-  ] ++ optional withXtables iptables
+  ] ++ optional withXtables iptables-nftables-compat
```

Comment:
This looks counter-intuitive to me.

Reply:
I think everything is correct.
The iptables package is used, which is compatible with the nftables syntax.

Reply:
but, why would someone want to use nftables with xtables support which also forwards to nftables?

Reply:
@zseri check out how `nft list ruleset` looks like without `withXtables` enabled: https://github.com/NixOS/nixpkgs/pull/81172#issuecomment-592500459